Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [workspace]
 members = [
-  "pages",
-  "tools/bencher",
-  "tools/formatter",
-  "tools/parser",
-  "tools/schema",
+	"pages",
```

Comment:
Can you change the indentation to four spaces instead of tabs? I'm not sure when this got set to two spaces.

Reply:
Ah, my formatter changed things. But adjusted it to 4 spaces then.